tripal_chado/api/tripal_chado.query.api.inc

```diff
@@ -1367,7 +1367,8 @@ function chado_select_record($table, $columns, $values, $options = NULL) {
 
       // CASE 1a: If there is only one element in the array, treat it the same
       // as a non-array value.
-      if (count($value) == 1 AND is_int(key($value))) {
+      if (count($value) == 1 AND is_int(key($value))
+          AND !(isset($value[0]['op']) && isset($value[0]['op']))) {
```

Comment:
I think this line has a duplicate?

```suggestion
          AND !isset($value[0]['op']) {
```

Reply:
Maybe you meant for it to be something else?

Reply:
Well, I wanted to also test 'data' and copy&paste but forgot to turn the second 'op' into 'data'. However, it (your correction) should work because we would expect to have a 'data' key when using a 'op' key.
I'll change 'op' into 'data' and it'll be fine. Thanks @almasaeed2010  for pointing out my mistake! :-)